README.md

```diff
@@ -267,6 +268,29 @@ As long as your are running locally, use the value `dev`.
 
 3. Save the .env file.
 
+
+## Adding Packages
+
+(Pipenv)[https://pipenv.pypa.io/en/latest/] is a packaging tool for Python that solves some common problems associated with the typical development workflow using pip, virtualenv, and the requirements.txt. In addition to addressing some common issues, it consolidates and simplifies the development process to a single command line tool.
+
+When installing new packages, you first need to assess if they are needed for production and development or only development.
+
+For packages needed for both prod and dev, please run installation as follows:
+
+```bash
+pipenv install <package-name>
+```
+
+If the package is only needed for development, please run installation as follows:
+
+```bash
+pipenv install <package-name> --dev
+```
+Providing the --dev argument will put the dependency in a special [dev-packages] location in the Pipfile. These development packages only get installed if you specify the --dev argument with pipenv install.
```

Comment:
What's the [] here for?

Reply:
That is how the section looks in the Pipfile. I feel like it is okay like this but if it caused a need for clarification, maybe I should update it to just say dev-package.

Reply:
Okay sounds good!